src/main/java/seedu/address/model/person/PersonContainsKeywordsPredicate.java

```diff
@@ -31,7 +31,6 @@ public boolean test(Person person) {
                 ||
                 keywords.stream()
                         .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getAddress().value, keyword));
-
     }
```

Comment:
The CI applications keep telling me that line 34 is 134 chars along, but it doesn't look like it to me. Any fixes?